sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -266,25 +371,21 @@ private static string GetBindingType(CustomAttribute attribute)
         {
             var attributeType = attribute.AttributeType.Name;
 
-            // TODO: fix this if we continue to use "<>EventAttribute" (questionable)
             // TODO: Should "webjob type" be a property of the "worker types" and come from there?
             return attributeType
                     .Replace("TriggerAttribute", "Trigger")
```

Comment:
Would be good to flag this here as eventually, it would be better to have attributes applied with this information so we're not based on convention only.

Reply:
Yeah I have the TODO above. I will create an issue for it.

Reply:
🎉

Reply:
https://github.com/Azure/azure-functions-dotnet-worker/issues/156